Tests/GDSCommonTests/ComponentTests/BulletViewTests.swift

```diff
@@ -7,7 +7,9 @@ internal final class BulletViewTests: XCTestCase {
     override func setUp() {
         super.setUp()
 
-        sut = .init(title: "exampleTitle", text: ["one", "two", "three"])
+        sut = .init(title: "exampleTitle",
```

Comment:
Could be worth testing the default value so with that in mind maybe worth creating 2 methods to return one with default and the other with a font set at init

Reply:
Done!